src/tools/build-manifest/src/main.rs

```diff
@@ -318,6 +318,7 @@ impl Builder {
         package!("rust-mingw", MINGW);
         package!("rust-std", TARGETS);
         self.package("rust-docs", &mut manifest.pkg, HOSTS, DOCS_FALLBACK);
+        self.package("rust-json-docs-preview", &mut manifest.pkg, HOSTS, DOCS_FALLBACK);
```

Comment:
Hm, so looking at this, can we rename the component to rust-docs-json-preview? I think keeping that common prefix will help with sorting the files etc and feels a little cleaner to me. I think we call them "JSON docs" in some places, so if you disagree I don't feel strongly at all about this and we can keep the current name (renaming it is more of a pain down the line).

Reply:
"JSON docs" feels more natural than "docs JSON", but I see your point. I don't feel strongly about either - @jyn514?

Reply:
The other 3-compenent name is `llvm-tools-preview`. The other two doc components are `rust-docs` and `rustc-docs`. `rustc-docs-json` and `rustc-json-docs` both seem like reasonable extensions to me, so I don't think there's an indicator there. But going from most specific seems good to me; `rust-json-docs` kind of sounds like documentation about save-analysis or something. So I have a slight preference for `rust-docs-json`.

Reply:
I've renamed everything to `rust-docs-json` and squashed all commits 👍🏻